script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -226,37 +225,37 @@ Just like Noel, this boy doesn't seem to fall under any definition of normal.
 [sha:7d01c82eb7ddd31fd6b67e3b6896b0aca6c59326]{
 -- Page 688, Offset 26118.
 -- 「んじゃあまあ、こっちから確認だな。
-"First things first, there's something I'd like to confirm.
+"Aight then, first I'd like to confirm somethin'.
 }
 [sha:31bc83badab4679f5660f73e93954c27109c393f]{
 -- Page 688, Offset 26119.
 -- オレたちの事はどこまで聞いている？」
-How much have you heard about us?"
+How much've heard 'bout us?"
 }
 [sha:3f7583ff8bea9b6a8110530ffd4d30e667ea77b4]{
 -- Page 688, Offset 26120.
 -- 「……吸血鬼退治の専門家だろ。アルクェイドは聖堂教会の代行者だと言っていた」
-"...You people specialize in slaying vampires, right?
+"You're vampire-slaying specialists, right?
  Arcueid said that you're 'Executors of the Holy Church'."
 }
 [sha:c0793e5b54edb3bc4177ae0c2a69a8051b929b1f]{
 -- Page 689, Offset 26121.
 -- 「そうか。それだけ知ってりゃ充分だ。説明が省ける。
-"Gotcha. Then that should be enough. I'll leave out the boring part.
+"Gotcha. Knowin' that's 'bout enuff. Don't gotta explain as much.
 }
 [sha:3426979555e0236b0ec0ad6c1f0b335a4dad98f9]{
 -- Page 689, Offset 26122.
 -- オレは親父の代わりに派遣された新人でな。実のところ、この国の事情はまだ半分ってところだ。
-I'm a new dispatch sent to represent my old man.
- I still barely have a handle on what's going on in this country.
+See, I was only dispatched here recently, standin' in for my dad.
+ If I'm bein' real, don't know most 'bout what's goin' in this country.
 }
 [sha:51b14479805b728dea6e21688b945ead57fb0b9b]{
 -- Page 689, Offset 26123.
 -- そこの三下はオレより早く調査していたがな。ただし独断。オレじゃなかったら師匠だけでなくテメェも更迭されてたからなブタ。感謝しろよ」
-That pipsqueak over there did some investigating before me.
- But she did it on her own.
- If it weren't for me, you'd have been dismissed along with your master, fatso.
- Show some gratitude."
+That shitter over there got a bit of investigation in before me, though.
```

Comment:
I think you suggested 'dipshit' somewhere else and that's better. "a shitter" is well, someone who sucks, I guess. 
Example: 
"2010, D. W. Chisholm, The Italian School Teacher (page 35)
Blood, guts, and hair were soon spread on the floor, and the dog was cut to pieces. Nat was muttering, “There, that'll teach you, you shitter,” as he sat down and looked at the bloodied remains of the animal."

That said, it might be too modern for the other kind of vocabulary I'm going for..